components/fonts/font.rs

```diff
@@ -193,8 +194,8 @@ impl FontMetrics {
 
 #[derive(Debug, Default)]
 struct CachedShapeData {
-    glyph_advances: HashMap<GlyphId, FractionalPixel>,
-    glyph_indices: HashMap<char, Option<GlyphId>>,
+    glyph_advances: FxHashMap<GlyphId, FractionalPixel>,
+    glyph_indices: FxHashMap<char, Option<GlyphId>>,
```

Comment:
One could still sent evil string of chars that will cause a lot of collisions, so I think we need to keep both on hashmap.